File: sdks/python/fiduscan/evidence.py

class EvidenceClient:
    def __init__(self, client):
        self.client = client

    def create_evidence(self, metadata):
        return self.client.request("POST", "/evidence", json=metadata)

    def verify_evidence(self, evidence_id):
        return self.client.request("GET", f"/evidence/{evidence_id}/verify")

File: sdks/node/src/modules/evidence.ts

import { Client } from "../client";

export class EvidenceClient {
  constructor(private client: Client) {}

  async createEvidence(metadata: any) {
    return this.client.request("POST", "/evidence", metadata);
  }

  async verifyEvidence(evidenceId: string) {
    return this.client.request("GET", `/evidence/${evidenceId}/verify`);
  }
}
